### PR TITLE
Fix issue #19 (edit command with no frames)

### PR DESCRIPTION
**Pull request by jmaupetit**

### CODE REVIEW

File: watson/cli.py [hunk 1]
```diff
@@ -478,6 +478,9 @@ def edit(watson, id):
     The `$EDITOR` environment variable is used to detect your editor.
     """
     if not id:
+        if not len(watson.frames):
+            click.echo("No frame to edit. It's time to start a new one!")
+            return
```
Comment (willdurand) [inline watson/cli.py]: Shouldn't we return an exit code here?

Comment (jmaupetit) [inline watson/cli.py]: I think click returns 0 when you return from a command, 1 when you raise, and 2 when it fails at parsing arguments. Do you think we should raise?

Comment (willdurand) [inline watson/cli.py]: Below, it raises an exception when the frame is not found. It is quite similar..

Comment (jmaupetit) [inline watson/cli.py]: You're right. Consistency is the key.

Comment (k4nar) [inline watson/cli.py]: Yep, we should raise a `click.ClickException` for that.

**Comment by jmaupetit** [comment]
Done.

**Pull request merged**